.github/workflows/build.yml

```diff
@@ -0,0 +1,242 @@
+name: build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/heads
+    branches:    
+      # Push events on main branch
+      - main
+
+env:
+  PKG_NAME: "vault"
+  GO_TAGS: "ui"
+
+jobs:
+  get-product-version:
+    runs-on: ubuntu-latest
```

Comment:
Why does the ent version use `runs-on: [linux, medium]` instead?

Reply:
Good question! `linux, medium` are tags for our private hosted runners, which enterprise uses. OSS uses github hosted runners

Reply:
ty